### PR TITLE
uses local cross-env

### PR DESCRIPTION
**Pull request by edm00se**
... instead of assuming global install, which it seems to be attempting to grab. There's likely a more graceful way of invoking it, such as creating it as an npm script of its own maybe, to prevent path specific invocation.

noglobalcrossenv

### CODE REVIEW

File: package-scripts.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ module.exports = {
     dev: {
       script: series([
         startInNewWindow('npm start dev.mongo --silent'),
-        startInNewWindow('cross-env PORT=8080 npm start dev.client --silent'),
+        startInNewWindow('./node_modules/.bin/cross-env PORT=8080 npm start dev.client --silent'),
```
Comment (kentcdodds) [inline package-scripts.js]: Perfect, thanks!

Comment (kentcdodds) [inline package-scripts.js]: ah, looks like this is too long. Try running `npm start format`

Also, you may wanna update your contributions in `.all-contributorsrc` to include `code` :)

Comment (kentcdodds) [inline package-scripts.js]: and run `npm start generate` :)

Comment (edm00se) [inline package-scripts.js]: Will do.

Comment (edm00se) [inline package-scripts.js]: It looks like `npm start format` only touches "api/**/*.js" & "client/**/*.js". I ran prettier-eslint against `package-scripts.js` directly and while it goes to a new line, it still fails eslint's 80-char rule.

File: package-scripts.js [hunk 2]
```diff
@@ -64,11 +64,14 @@ module.exports = {
     dev: {
       script: series([
         startInNewWindow('npm start dev.mongo --silent'),
-        startInNewWindow('cross-env PORT=8080 npm start dev.client --silent'),
+        startInNewWindow(
+          './node_modules/.bin/cross-env PORT=8080 ' +
+            'npm start dev.client --silent'
```
Comment (edm00se) [inline package-scripts.js]: It feels a little silly to resort to string concatenation to pass an eslint rule, but ¯\_(ツ)_/¯

**Comment by edm00se** [comment]
You are quite welcome.

File: package-scripts.js (hunk 2 above)
Comment (kentcdodds) [inline package-scripts.js]: Oh, I should have mentioned, use `oneLine` from [`common-tags`](https://github.com/declandewet/common-tags)! Woo!

Comment (edm00se) [inline package-scripts.js]: Even better!

**Comment by kentcdodds** [comment]
Thanks!

**Comment by edm00se** [comment]
My pleasure.

**Pull request merged**